themes/scss/widgets/form-elements.scss

```diff
@@ -7,17 +7,15 @@
 .CDB-Widget-textInput {
   border: none;
   outline: none;
-  font-family: 'Open Sans';
-  font-size: $sFontSize-large;
 }
 .CDB-Widget-textInput::-webkit-input-placeholder {
-  font-weight: $sFontWeight-lighter;
+  font-weight: 300;
```

Comment:
Don't we already have a variable for that?
Also, didn't we say that 300 was too light?